snippets/conserv_with_d3.json

```diff
@@ -0,0 +1,3 @@
+{
+  "js": ["/snippets/includes/d3.v3.5.17.min.js"]
```

Comment:
better install d3 as npm dependency and link to the npm folder. we also have "buildJS" for the web, so we can then provide a cdn version too.